File: official/docs/php/v6/billing/create-ep-credit-card.php
<?php

$client = new \EasyPost\EasyPostClient('EASYPOST_API_KEY');

$referralUserApiKey = $_ENV['REFERRAL_USER_PROD_API_KEY'];

$creditCard = $client->referralCustomer->addCreditCard(
    $referralUserApiKey,
    '0123456789101234',
    '01',
    '2025',
    '111'
);

echo $creditCard;

File: official/docs/python/v8/billing/create-ep-credit-card.py
import easypost


client = easypost.EasyPostClient("EASYPOST_API_KEY")

credit_card = client.referral_customer.add_credit_card(
    referral_api_key="REFERRAL_USER_API_KEY",
    number="0123456789101234",
    expiration_month="01",
    expiration_year="2025",
    cvc="111",
)

print(credit_card)
